src/dotty/tools/dotc/core/OrderingConstraint.scala

```diff
@@ -0,0 +1,502 @@
+package dotty.tools
+package dotc
+package core
+
+import Types._, Contexts._, Symbols._, Decorators._
+import util.SimpleMap
+import collection.mutable
+import printing.{Printer, Showable}
+import printing.Texts._
+import config.Config
+import config.Printers._
+import collection.immutable.BitSet
+import reflect.ClassTag
+
+object OrderingConstraint {
+
+  /** The type of `Constraint#boundsMap` */
```

Comment:
"The type of `OrderingConstraint#boundsMap`" would be more correct, same thing for `ParamOrdering`.